scripts/all-tests.sh

```diff
@@ -137,8 +137,18 @@ END
                 paver test_acceptance -s lms --extra_args="-v 3"
                 ;;
 
+            "2")
+                mkdir -p reports
+                mkdir -p reports/acceptance
+                cat > reports/acceptance/xunit.xml <<END
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuite name="nosetests" tests="1" errors="0" failures="0" skip="0">
+<testcase classname="lettuce.tests" name="shard_placeholder" time="0.001"></testcase>
+</testsuite>
+END
+                ;;
             *)
-                paver test_acceptance -s lms --extra_args="-v 3 --tag shard_${SHARD}"
```

Comment:
Actually in this case, if someone has not rebased, then the tests will only run shard_1, assuming we change Jenkins config. So I suggest we keep Jenkins config for a bit longer (1-2 weeks), then announce it as a breaking config change. That way, for the intermediate period, older commits will run both shards like always, and newer commits will have a very, very short shard_2.